### ISSUE TITLE
Unable to setup Ammonite scripts on VSCode

### ISSUE THREAD

**Issue by tusharmath** [scalameta/metals#2526]
**To Reproduce**

1. Create a new file `Foo.sc` 
   ```scala
   // Foo.sc
   object Route {
      val a = 1
   }
   ```
2. Open the file in VSCode with Metals installed.
3. Click on "Import Ammonite scripts automatically"
  <img width="489" alt="Screenshot 2021-02-19 at 10 54 29 AM" src="https://user-images.githubusercontent.com/194482/108461525-db07fe00-72a0-11eb-9fbf-c060b58b1e1c.png">

**Expected behavior**

- Code should compile

**Screenshots**
<img width="491" alt="Screenshot 2021-02-19 at 10 51 13 AM" src="https://user-images.githubusercontent.com/194482/108461270-646b0080-72a0-11eb-905e-c72e9fb016e3.png">

**Installation:**

- Operating system: macOS/Windows/Linux
- Editor: Visual Studio Code/Atom/Vim/Sublime/Emacs
- Metals version: v0.9.10

**Additional context**

**metals.log**
```stack
ammrunner.CoursierError: Error fetching Ammonite 2.3.8-4-88785969 for scala 2.12.13
	at ammrunner.AmmoniteFetcher.maybeResult$1(AmmoniteFetcher.scala:73)
	at ammrunner.AmmoniteFetcher.command(AmmoniteFetcher.scala:91)
	at scala.meta.internal.metals.ammonite.Ammonite.command(Ammonite.scala:159)
	at scala.meta.internal.metals.ammonite.Ammonite.$anonfun$start$4(Ammonite.scala:237)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.ammonite.Ammonite.start(Ammonite.scala:237)
	at scala.meta.internal.metals.ammonite.Ammonite.doImport$1(Ammonite.scala:188)
	at scala.meta.internal.metals.ammonite.Ammonite.$anonfun$maybeImport$3(Ammonite.scala:213)
	at scala.meta.internal.metals.ammonite.Ammonite.$anonfun$maybeImport$3$adapted(Ammonite.scala:206)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.lang.Thread.run(Thread.java:834)
Caused by: coursierapi.error.SimpleResolutionError$1: Error downloading com.lihaoyi:ammonite-repl-api_2.12.13:2.3.8-4-88785969
  not found: /Users/tushar/.ivy2/local/com.lihaoyi/ammonite-repl-api_2.12.13/2.3.8-4-88785969/ivys/ivy.xml
  not found: https://repo1.maven.org/maven2/com/lihaoyi/ammonite-repl-api_2.12.13/2.3.8-4-88785969/ammonite-repl-api_2.12.13-2.3.8-4-88785969.pom
	at coursierapi.error.SimpleResolutionError.of(SimpleResolutionError.java:11)
	at coursierapi.shaded.coursier.internal.api.ApiHelper$.simpleResError(ApiHelper.scala:326)
	at coursierapi.shaded.coursier.internal.api.ApiHelper$.doFetch(ApiHelper.scala:359)
	at coursierapi.shaded.coursier.internal.api.ApiHelper.doFetch(ApiHelper.scala)
	at coursierapi.Fetch.fetchResult(Fetch.java:244)
	at ammrunner.AmmoniteFetcher.maybeResult$1(AmmoniteFetcher.scala:70)
	at ammrunner.AmmoniteFetcher.command(AmmoniteFetcher.scala:91)
	at scala.meta.internal.metals.ammonite.Ammonite.command(Ammonite.scala:159)
	at scala.meta.internal.metals.ammonite.Ammonite.$anonfun$start$4(Ammonite.scala:237)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.ammonite.Ammonite.start(Ammonite.scala:237)
	at scala.meta.internal.metals.ammonite.Ammonite.doImport$1(Ammonite.scala:188)
	at scala.meta.internal.metals.ammonite.Ammonite.$anonfun$maybeImport$3(Ammonite.scala:213)
	at scala.meta.internal.metals.ammonite.Ammonite.$anonfun$maybeImport$3$adapted(Ammonite.scala:206)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.lang.Thread.run(Thread.java:834)
```

**Comment by ckipp01** [comment]
Thanks for the report @tusharmath! You actually stumbled on a bug in the last release where for Ammonite support we default to 2.12.13, but the default version of Ammonite that you see there isn't actually available in 2.12.13. There are two ways to get around this:

1. At the top of your file use a different Scala or Ammonite version like you can [see here](https://scalameta.org/metals/docs/troubleshooting/faq.html#how-do-i-use-scala-2xx-for-my-script)
2. Use the nightly as it's fixed in there (we are planning a release very soon as well)

**Comment by tusharmath** [comment]
Thanks you so much!
